src/main/java/bio/terra/service/job/StairwayLoggingHooks.java

```diff
@@ -51,7 +52,8 @@ public HookAction endFlight(FlightContext context) {
     @Override
     public HookAction endStep(FlightContext context) {
         logger.info(StepLogFormat, "endStep", context.getFlightClassName(), context.getFlightId(),
-            context.getStepIndex(), Instant.now().atZone(ZoneId.of("Z")).format(DateTimeFormatter.ISO_INSTANT));
+            context.getStepClassName(), context.getStepIndex(), context.getDirection().name(),
+            Instant.now().atZone(ZoneId.of("Z")).format(DateTimeFormatter.ISO_INSTANT));
```

Comment:
What is your thought behind adding a timestamp? Wouldn't the stackdriver logs capture this?

Reply:
I didn't add the timestamp. I _think_ the reason it is there is because the time the log message is generated can be different than the time it hits the google log, but I am not sure.

Reply:
Ah, my mistake. Thanks for the clarification!